applications/RANSApplication/custom_python/add_custom_utilities_to_python.cpp

```diff
@@ -29,14 +30,34 @@ void AddCustomUtilitiesToPython(pybind11::module& m)
+    py::class_<RansScalarVariableDifferenceNormCalculationUtilityType, RansScalarVariableDifferenceNormCalculationUtilityType::Pointer>(m, "ScalarVariableDifferenceNormCalculationUtility")
+        .def(py::init<const ModelPart&, const Variable<double>&>())
+        .def("InitializeCalculation", &RansScalarVariableDifferenceNormCalculationUtilityType::InitializeCalculation)
+        .def("CalculateDifferenceNorm", &RansScalarVariableDifferenceNormCalculationUtilityType::CalculateDifferenceNorm);
+
     m.def_submodule("RansVariableUtilities")
         .def("ClipScalarVariable", &RansVariableUtilities::ClipScalarVariable)
         .def("GetMinimumScalarValue", &RansVariableUtilities::GetMinimumScalarValue)
         .def("GetMaximumScalarValue", &RansVariableUtilities::GetMaximumScalarValue)
-        .def("CopyNodalSolutionStepVariablesList", &RansVariableUtilities::CopyNodalSolutionStepVariablesList);
+        .def("AddAnalysisStep", &RansVariableUtilities::AddAnalysisStep)
+        .def("IsAnalysisStepCompleted", &RansVariableUtilities::IsAnalysisStepCompleted)
+        .def("AssignConditionVariableValuesToNodes", &RansVariableUtilities::AssignConditionVariableValuesToNodes<double>, py::arg("model_part"), py::arg("variable"), py::arg("flag"), py::arg("flag_value") = true)
+        .def("AssignConditionVariableValuesToNodes", &RansVariableUtilities::AssignConditionVariableValuesToNodes<array_1d<double, 3>>, py::arg("model_part"), py::arg("variable"), py::arg("flag"), py::arg("flag_value") = true)
+        .def("AssignBoundaryFlagsToGeometries", &RansVariableUtilities::AssignBoundaryFlagsToGeometries)
+        .def("FixFlaggedDofs", &RansVariableUtilities::FixFlaggedDofs, py::arg("model_part"), py::arg("variable"), py::arg("check_flag"), py::arg("check_value") = true)
+        .def("CopyNodalSolutionStepVariablesList", &RansVariableUtilities::CopyNodalSolutionStepVariablesList)
+        .def("CopyNodalFlaggedVariableFromHistoricalToNonHistorical", &RansVariableUtilities::CopyNodalFlaggedVariableFromHistoricalToNonHistorical<double>, py::arg("model_part"), py::arg("variable"), py::arg("check_flag"), py::arg("check_value") = true)
+        .def("CopyNodalFlaggedVariableFromHistoricalToNonHistorical", &RansVariableUtilities::CopyNodalFlaggedVariableFromHistoricalToNonHistorical<array_1d<double, 3>>, py::arg("model_part"), py::arg("variable"), py::arg("check_flag"), py::arg("check_value") = true)
+        .def("CopyNodalFlaggedVariableFromNonHistoricalToHistorical", &RansVariableUtilities::CopyNodalFlaggedVariableFromNonHistoricalToHistorical<double>, py::arg("model_part"), py::arg("variable"), py::arg("check_flag"), py::arg("check_value") = true)
+        .def("CopyNodalFlaggedVariableFromNonHistoricalToHistorical", &RansVariableUtilities::CopyNodalFlaggedVariableFromNonHistoricalToHistorical<array_1d<double, 3>>, py::arg("model_part"), py::arg("variable"), py::arg("check_flag"), py::arg("check_value") = true)
```

Comment:
Should the arguments of all these functions be `*args`? I have the feeling that some of the must be always provided...